src/Dapr.Client/DaprClientGrpc.cs

```diff
@@ -1397,10 +1397,10 @@ private Autogenerated.StateItem ToAutogeneratedStateItem(StateTransactionRequest
                 InstanceId = instanceId,
                 WorkflowComponent = workflowComponent,
                 WorkflowName = workflowType,
-                Input = input
+                Input = (ByteString)input
```

Comment:
Yikes, this is not what I had in mind when I suggested allowing `object` instead of `ByteString`. What you have here seems very problematic because it will result in `InvalidCastException` whenever a user specifies an input that can't be cast directly into `ByteString`. What I was actually proposing is that we accept any object, [JSON-serialize it](https://learn.microsoft.com/en-us/dotnet/standard/serialization/system-text-json/how-to?pivots=dotnet-core-3-1), and then convert the serialized input JSON into whatever the protobuf contract requires (`ByteString`, in this case).